src/coreclr-debug/activate.ts

```diff
@@ -43,6 +43,9 @@ async function checkForInvalidArchitecture(platformInformation: PlatformInformat
         else if (platformInformation.architecture !== "x86_64") {
             if (platformInformation.isWindows() && platformInformation.architecture === "x86") {
                 eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: x86 Windows is not currently supported by the .NET Core debugger. Debugging will not be available.`));
+            } else if (platformInformation.isMacOS() && platformInformation.architecture === "arm64") {
```

Comment:
This isn't enough - the debugger will still not download. You can likely address that by adding the following to package.json:

```jsonc
    {
      "id": "Debugger",
      "description": ".NET Core Debugger (macOS / x64)",
      "url": "https://download.visualstudio.microsoft.com/download/pr/6f481c2a-74a8-41cc-a115-f2d059242062/96e3d82b95ff1375347fedf0f5c7d4e7/coreclr-debug-osx-x64.zip",
      "fallbackUrl": "https://vsdebugger.blob.core.windows.net/coreclr-debug-1-23-8/coreclr-debug-osx-x64.zip",
      "installPath": ".debugger",
      "platforms": [
        "darwin"
      ],
      // Line 275 of package.json -- Add 'arm64'
      "architectures": [
        "x86_64",
        "arm64"
      ],
      "binaries": [
        "./vsdbg-ui",
        "./vsdbg"
      ],
      "installTestPath": "./.debugger/vsdbg-ui",
      "integrity": "6BAA9A20DAA71018B9912A3E2A48C52A273ADC2E70F7101C12F688F397AD16D8"
    },
```

Reply:
Alright, fixed that.